engine/statmetrics.go

```diff
@@ -74,6 +74,7 @@ type StatMetric interface {
 	GetStringValue(roundingDecimal int) (val string)
 	GetFloat64Value(roundingDecimal int) (val float64)
 	AddEvent(evID string, ev utils.DataProvider) error
+	OneEvent(ev utils.DataProvider) error
```

Comment:
Since all the other methods are having also the verb attached, please rename the method to AddOneEvent